Tests/PackageGraphPerformanceTests/DependencyResolverPerfTests.swift

```diff
@@ -174,7 +174,7 @@ extension ProductFilter: JSONSerializable, JSONMappable {
         switch self {
         case .everything:
             return "all".toJSON()
-        case .specific(let products):
+        case .specific(let products, _):
```

Comment:
Oops. I forgot to come back to this. I will look at the performance tests tomorrow.

Reply:
These performance tests are inoperable on `main` anyway, because the JSON files are out of date.